internal/commands/user/disable.go

```diff
@@ -47,12 +47,17 @@ func (cmd *CommandDisable) Handler(profile *cli.Profile, ui terminal.UI) error {
 		return appErr
 	}
 
-	users, usersErr := cmd.realmClient.FindUsers(app.GroupID, app.ID, realm.UserFilter{IDs: cmd.inputs.Users})
-	if usersErr != nil {
-		return usersErr
+	resolvedUsers, resolveErr := cmd.inputs.resolveUsers(cmd.realmClient, app.GroupID, app.ID)
```

Comment:
[nit] consistent prefix

Reply:
[optional] to add to this, maybe just stick to `resolved`/`selected` for the names of the user arrays?